webrtc.html

```diff
@@ -3119,6 +3119,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             be set to that track and the MSID and media type generated
             by <code>createOffer</code> will be that of the track.</p>
 
+            <p>If a kind is passed in and the value is not <code>audio</code>
+            or <code>video</code>, throw a <code>TypeError</code>.</p>
```

Comment:
Can we refer to "a legal value of the track.kind" or something like that instead of hard-coding "audio & video"?